content/lessons/js2/sublesson/map_reduce_and_filter_functions.mdx

```diff
@@ -1985,16 +1985,6 @@ const friends = ['Tony Stark', 'Vision', 'Ultron'].reduce((a, b) => {
 
 </Spoiler>
 
-```jsx
-const friends = ['Tony Stark', 'Vision', 'Ultron'].reduce((a, b) => {
-  setTimeout(() => {
-    console.log('Greetings,', b)
-  }, 2000)
-  return a + b
-}, '') // What is friends?
-
-// What gets printed out into the console?
-```
```

Comment:
I think you might have deleted the wrong code block. Did you mean to delete the code block that's inside the `<Spoiler>` section above instead of this code block?

Reply:
You're absolutely right! It should be fixed now.